modules/microshift-greenboot-check-boot.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * microshift_running applications/microshift-greenboot.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-greenboot-check-boot_{context}"]
+= Checking an update's success with greenboot
+
+After an update, you can determine whether the update was declared valid by checking the boot. Such a health check can help you determine if you the system is working properly and you can proceed without a rollback.
```

Comment:
All these processes are automatic, so using "you" may not be appropriate here. 

Also: "by checking the boot" needs to be fixed. Do you mean "greenboot health check status"?